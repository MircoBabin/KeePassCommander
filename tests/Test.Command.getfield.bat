@echo off
    setlocal
    call "%~dp0Utils\Utils.test-setup.bat" "%~f0"
    
    "%KeePassCommandExe%" getfield "Sample Entry" "extra field 1" "extra password 1"> "%KeePassCommand_TestOutput%"
    
    call "%~dp0Utils\Utils.diff.bat"
    if errorlevel 1 exit /b 1
    exit /b 0    