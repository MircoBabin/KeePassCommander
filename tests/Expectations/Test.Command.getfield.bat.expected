SUCCESS
B	
I	title	Sample Entry
I	extra field 1	U29tZSBleHRyYSBkYXRh
I	extra password 1	U29tZSBleHRyYSBwYXNzd29yZA==
E	

